files/default/lib/bumpzone.rb

```diff
@@ -0,0 +1,115 @@
+#!/opt/chef/embedded/bin/ruby
+require 'git'
+require 'json'
+require 'octokit'
+
+# Library to bump DNS zones
+class BumpZone
+  @commit_msg = "\n\n"
+  @warning_msg = "Warning:\n\n"
+  @warning = false
+  ONELINE_REGEX = /([\w\d\.-_]+\s+[\d]+\s+IN\s+SOA.*[\w\d\.-_]+\s+)(\d{10})(\s+[\d]+\s+[\d]+\s+[\d]+\s+[\d]+)/
+  MULTILINE_REGEX = /([\w\d\.-_]+\s+IN\s+SOA\s+[\w\d\.-_]+\s+[\w\d\.-_]+\s+\(\n\s+)(\d{10})(.*\n.*\n.*\n.*\n.*\n\s+\))/
```

Comment:
@jldugger great catch on those. I'm working on fixing that now. Are there any other situations you can think of? I removed underscores as I realized they're not allowed in RFC.